rocAL/rocAL/source/augmentations/color_augmentations/node_brightness.cpp

```diff
@@ -25,43 +25,45 @@ THE SOFTWARE.
 #include "exception.h"
 
 
-BrightnessNode::BrightnessNode(const std::vector<Image *> &inputs, const std::vector<Image *> &outputs) :
+BrightnessNode::BrightnessNode(const std::vector<rocalTensor *> &inputs, const std::vector<rocalTensor *> &outputs) :
         Node(inputs, outputs),
         _alpha(ALPHA_RANGE[0], ALPHA_RANGE[1]),
         _beta (BETA_RANGE[0], BETA_RANGE[1])
 {
 }
 
-void BrightnessNode::create_node()
-{
+void BrightnessNode::create_node() {
     if(_node)
         return;
 
     _alpha.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
     _beta.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
 
-    _node = vxExtrppNode_BrightnessbatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), _batch_size);/*A temporary fix for time being*/
+    int input_layout = (int)_inputs[0]->info().layout();
```

Comment:
If input_layout, output_layout & roi_type is accessed in all nodes can we add this as part of node. So by default it will be there when we do the inheritance. 
We can assign these values during node creation itself.

Reply:
Done